tasks/syntaxPractice.go

```diff
@@ -1,31 +1,52 @@
 package tasks
 
+import "fmt"
+
 // Task 1: Write a function that prints "Hello World" to the console.
 
 func PrintHelloWorld() {
+	fmt.Println("Hello World!")
+
 	// Implement the code to print "Hello World"
 }
 
 // Task 2: Write a function that takes an integer and prints it to the console using string formatting.
-
 func PrintInteger(n int) {
+	var num int
+	fmt.Scanln(&num)
+	fmt.Println(num)
 	// Implement the code to print the integer n.
```

Comment:
No need to take input for num, just implement the code to print the int n passed as an argument to the function.